src/app/data-planes/components/BuiltinGatewayPolicies.vue

```diff
@@ -50,7 +61,18 @@
                     <div class="dataplane-policy-header">
                       <div>
                         <div>
-                          <b>Route</b>: <router-link :to="routeEntry.route">{{ routeEntry.routeName }}</router-link>
+                          <b>Route</b>: <RouterLink
+                            :to="{
+                              name: 'policy-detail-view',
+                              params: {
+                                mesh: routeEntry.route.mesh,
+                                policyPath: props.policyTypesByName[routeEntry.route.type]!.path,
```

Comment:
How come the `!` here?

Reply:
This used to use the assertion `as PolicyType` only tell TypeScript that this can’t be `undefined`. Using `!` is producing the same result type-wise and is simpler